pkg/sdn/api/validation/validation.go

```diff
@@ -86,7 +86,7 @@ func ValidateNetNamespace(netnamespace *sdnapi.NetNamespace) fielderrors.Validat
 	allErrs := fielderrors.ValidationErrorList{}
 	allErrs = append(allErrs, validation.ValidateObjectMeta(&netnamespace.ObjectMeta, false, oapi.MinimalNameRequirements).Prefix("metadata")...)
 
-	if netnamespace.NetID < 0 {
+	if netnamespace.NetID != nil && *netnamespace.NetID < 0 {
```

Comment:
validate it is in the expected range `0 <= x <= 2^24-1... conversion should ensure that the special KnownInvalid value gets converted to nil, so anything outside that range we need to return a validation error on, right?